packages/components/collapsible/src/Collapsible.test.tsx

```diff
@@ -0,0 +1,151 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { useState } from 'react'
+import { describe, expect, it, vi } from 'vitest'
+
+import { Collapsible } from '.'
+
+describe('Collapsible', () => {
+  it('should open the collapsible content when clicking on the trigger', async () => {
+    const user = userEvent.setup()
+
+    render(
+      <Collapsible>
+        <Collapsible.Trigger asChild>
+          <button>Expand</button>
```

Comment:
Yes, I did it like that because it real use-case implementation `asChild` will always be used (as `Collapsible.Trigger` will wrap a Spark Button most of the time, or a link)

Reply:
Completely agree but IMHO we also need to verify that technically our component will behave also as expected without this prop (even if it will be used most of the time).

Reply:
Ok i remove it from the base test that open and close the Collapsible to cover this.